src/librustc_resolve/lib.rs

```diff
@@ -4169,7 +4169,7 @@ impl<'a, 'crateloader: 'a> Resolver<'a, 'crateloader> {
                                           lookup_name: Name,
                                           namespace: Namespace,
                                           start_module: &'a ModuleData<'a>,
-                                          graph_root: bool,
+                                          name: Name,
```

Comment:
Maybe this should be an `Ident`, actually? The idea is that it is the thing we are going to put at the front of the path

Reply:
Also, let's give it a descriptive name, like `crate_name`